docker-compose.yml

```diff
@@ -6,7 +6,7 @@ db:
   environment:
     - MYSQL_ROOT_PASSWORD=my-super-secret-password
 web:
-  image: openxpki:latest
+  build: ./
```

Comment:
This should be for dev/test environments only, right?

Reply:
Oh yes, my mistake sorry.
```suggestion
  + image: openxpki:latest
  - build: ./
```

Reply:
No worries, looks good.